WooCommerce/Classes/ViewRelated/Products/Categories/Selector/ProductCategorySelector.swift

```diff
@@ -0,0 +1,89 @@
+import SwiftUI
+
+/// View showing a list of product categories to select from.
+///
+struct ProductCategorySelector: View {
```

Comment:
TBH we don't really have coding convention for SwiftUI views yet. I believe we are following Apple to only add the `View` suffix to the SwifUI views only when it's absolutely necessary. E.g: they have `VStack` instead of VStackView, and they also have `ContentView` since apparently `Content` sounds misleading.

In this case, I believe `Selector` is sufficient - we can understand that it's a view to select items. But please feel free to start a discussion in our team channel, since it's best that we keep a consistent naming pattern among the team and update the coding convention guide accordingly.

Reply:
I understand. Thanks for the explanation.